.github/workflows/devbuild.yml

```diff
@@ -72,6 +72,13 @@ jobs:
           echo "flake8 path: $(which flake8)"
           echo "flake8 version: $(flake8 --version)"
 
+      - name: ccache
```

Comment:
The job standalone_buffer runs fairly fast.  If it uses [ccache](https://ccache.dev), the PR implies all building jobs is supposed to use ccache.  Is my assumption correct?

Is it a common practice to apply ccache to all building jobs in Github Actions?